crates/rattler/src/cli/auth.rs

```diff
@@ -110,6 +130,18 @@ fn login(args: LoginArgs, storage: AuthenticationStorage) -> Result<(), Authenti
         }
     } else if let Some(token) = args.token {
         Authentication::BearerToken(token)
+    } else if let Some(access_key_id) = args.s3_access_key_id {
```

Comment:
```suggestion
    } else if let (Some(access_key_id), Some(secret_access_key)) = (args.s3_access_key_id, args.s3_secret_access_key) {
```

maybe like this @pavelzw? It's a bit asymmetric right now that we raise a dedicated error for `access_key_id != None && secret_access_key == None` and not for `secret_access_key != None && access_key_id == None`. This way, we wouldn't raise a custom error, but just fall back to `AuthenticationCLIError::NoAuthenticationMethod`.